main.go

```diff
@@ -28,6 +29,10 @@ func getTerraformDependencies(folder string) (map[string]*tfconfig.Module, error
 	modules[folder] = module
 
 	for _, res := range module.ModuleCalls {
+		// pick only modules with source in local paths
+		if !strings.HasPrefix(res.Source, "../") && !strings.HasPrefix(res.Source, "./") {
```

Comment:
```suggestion
		if !tfconfig.IsModuleDir(res.Source) {
```

Let's use `tfconfig` function instead of depending on module path.